TicTacToe.hpp

```diff
@@ -30,9 +30,14 @@ class TicTacToe
 		// These functions should only be called from within this class.
 		void changeTurn(); //Phillip
 		void checkVictory();
+		// Below three functions will be called within checkVictory(); take teamID as parameter to count score for that team; return socre.
+		int checkVictoryHorizontal(int teamSide); // check all (18) horizontal rows; 9 for left to right, 9 for front to back.
+		int checkVictoryVertical(int teamSide); // check all (9) vertical rows; 9 for bottom to top.
+		int checkVictoryDiagonal(int teamSide); // check all (22) diagonal rows; 6 6 6 4 (I don't know how to describe them with words).
 
 		// Stores the state of the game board
-		// * Index 0 is the bottom and index 2 is top of each stack.
+		// For each deque<int>, index 0 is the bottom and index 2 is top of each stack.
```

Comment:
Could you keep the formatting and preface the list items with `*` like I did? I think it makes the multi-line comments more clear to read.

Reply:
I added * in front of both of the lines. I also merged a line below.